native/cocos/audio/android/AudioDecoderSLES.h

```diff
@@ -43,7 +44,7 @@ class AudioDecoderSLES : public AudioDecoder {
     void queryAudioInfo();
 
     void signalEos();
-    void decodeToPcmCallback(SLAndroidSimpleBufferQueueItf queueItf);
+    void decodeToPcmCallback(BufferQueueItf queueItf);
```

Comment:
Is `BufferQueueItf` valid on Android?

Reply:
Use typedef to define bufferqueueitf, Android uses slandroidsimplebufferqueueitf, and openharmony uses other